content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md

```diff
@@ -1,5 +1,5 @@
 
-Run this to revert any changes made to this host by 'kubeadm init' or 'kubeadm join'
+Performs a best effort revert of changes made to this host by 'kubeadm init' or 'kubeadm join'
```

Comment:
@Klaven the change needs to be in https://github.com/kubernetes/website/blob/master/content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md#reset-workflow-reset-workflow and not the generated file.

Reply:
:man_facepalming: fixed